jib-core/src/test/java/com/google/cloud/tools/jib/frontend/CredentialRetrieverFactoryTest.java

```diff
@@ -60,61 +59,57 @@ public void setUp()
   }
 
   @Test
-  public void testDockerCredentialHelper() throws Exception {
+  public void testDockerCredentialHelper() throws CredentialRetrievalException {
     CredentialRetrieverFactory credentialRetrieverFactory =
         CredentialRetrieverFactory.forImage(
             ImageReference.of("registry", null, null), mockJibLogger);
-
-    Mockito.when(
-            mockDockerCredentialHelperFactory.newDockerCredentialHelper(
-                "registry", Paths.get("docker-credential-helper")))
-        .thenReturn(mockDockerCredentialHelper);
+    credentialRetrieverFactory.setDockerCredentialHelperFactory(
+        (registry, credentialHelper) -> mockDockerCredentialHelper);
```

Comment:
Should check that the parameters passed in are what was expected. Ditto everywhere the `Mockito.when` is removed.